components/automate-ui/src/app/pages/+compliance/shared/reporting/report-query.service.ts

```diff
@@ -1,83 +1,65 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
-import { FilterC } from '../../+reporting/types';
-import { find, uniqWith, isEqual } from 'lodash';
 import * as moment from 'moment';
+import { FilterC } from '../../+reporting/types';
```

Comment:
These changes mostly add the date interval index, start date, and end date to the BehaviorSubject with the search bar filters. This allows all the pages to be notified when all these fields are updated. 

This is also preparing to move this code to NGRX/store.